diesel/src/macros/mod.rs

```diff
@@ -174,36 +175,44 @@ macro_rules! __diesel_column {
 #[macro_export]
 macro_rules! table {
     (
-        $name:ident {
-            $($column_name:ident -> $Type:ty,)+
+        $name:ident $body:tt
+    ) => {
+        table! {
+            public . $name (id) $body
```

Comment:
~~TIL~~ TIR (remembered): In macro token trees, whitespace around `.` and tokens don't matter (the same way they don't matter around `+`). Still reads weird though ;)